lib/client/ood.js

```diff
@@ -154,7 +154,15 @@ function download() {
             if (isDir)
                 path	    = CloudCmd.PREFIX + '/oodzip' + path;
             else
-                path        = prefixUr + FS + path + '?download';
+                // Workaround for large files filling up /tmp folder
+                // uses a custom nginx route configured at /pun/download
+                if (ood_download) {
+                    path        = '/pun/download' + path + '?download=' + Date.now();
+                } else {
+                    path        = prefixUr + FS + path + '?download';
```

Comment:
This will never get hit. `ood_download` will always exist, even if it is `ood_download: ""`